sockeye/transformer.py

```diff
@@ -102,7 +85,7 @@ def __init__(self,
                                                    prefix="ff_post_",
                                                    num_hidden=config.model_size)
             self.lhuc = None
-            if config.use_lhuc:
+            if config.lhuc:
```

Comment:
Renaming this wouldn't be backwards compatible, right? I would leave it at `use_lhuc`.

Reply:
good catch, will revert